source/src/molecular-unfolding/cdk/stack-main.ts

```diff
@@ -76,6 +76,14 @@ export class MainStack extends SolutionStack {
       constraintDescription: 'Any printable ASCII character ranging from the space character (\u0020) through the end of the ASCII character range',
     });
 
+    const quickSightRoleArn = new CfnParameter(this, 'QuickSightRoleArn', {
```

Comment:
might be it'd better using role name without including account id

Reply:
change to only use role name